src/index.js

```diff
@@ -26,7 +26,7 @@ module.exports = (config) => {
     //  time between two reconnect (ms)
     timeout: 1000,
     //  default timeout for RPC calls. If set to '0' there will be none.
-    rpcTimeout: 1000,
+    rpcTimeout: 15000,
```

Comment:
I think the idea is to override it in projects but give generous time per default.